alerting/src/alert-worker.ts

```diff
@@ -4,7 +4,11 @@ import { ANALYSIS_API_URL } from './constants';
 import { Alert } from './entities/alerts.entity';
 import { calculateBoundsForAlert } from './utils/analysis-utils';
 import { sendEmail } from './utils/email';
-import { getWCSCoverage, getWMSCapabilities } from './utils/server-utils';
+import {
+  formatUrl,
+  getWCSCoverage,
+  getWMSCapabilities,
+} from './utils/server-utils';
 
 async function processAlert(alert: Alert, alertRepository: Repository<Alert>) {
   const { baseUrl, serverLayerName, type } = alert.alertConfig;
```

Comment:
I think hazardLayerIds should come from the `id` in alertConfig, and not serverLayerName, which is the representation we use to query the data.